src/AppInstallerRepositoryCore/ArpVersionValidation.cpp

```diff
@@ -67,11 +67,10 @@ namespace AppInstaller::Repository
             {
                 if (manifestArpVersionRange.Overlaps(arpInIndex))
                 {
-                    std::string errorMsg = Manifest::ManifestError::ArpVersionOverlapWithIndex;
-                    errorMsg.append("[" + arpInIndex.GetMinVersion().ToString() + ", " + arpInIndex.GetMaxVersion().ToString() + "]");
-                    AICLI_LOG(Repo, Error, << errorMsg);
+                    std::string errorMsg = ("[" + arpInIndex.GetMinVersion().ToString() + ", " + arpInIndex.GetMaxVersion().ToString() + "]");
+                    AICLI_LOG(Repo, Error, << Utility::ConvertToUTF8(Manifest::ManifestError::ArpVersionOverlapWithIndex) << errorMsg);
                     THROW_EXCEPTION(Manifest::ManifestException(
-                        { Manifest::ValidationError(errorMsg) },
+                        { Manifest::ValidationError(Manifest::ManifestError::ArpVersionOverlapWithIndex, errorMsg) },

```

Comment:
This is calling the constructor `ValidationError(AppInstaller::StringResource::StringId message, std::string field)`. That doesn't seem right to me as this is not a manifest field (unless I'm misunderstanding what `field` means here).

Reply:
I suggested that it was now "context" rather than "field".  Would be good to change the name to reflect that.